Automattic/jetpack
# Publicize: Add icon fallback if profile picture fails to load

In some instances, a profile picture for a Publicize connection can fail to load, for example when the image URL is expired or gets blocked by a browser extension. This PR introduces a change that handles these failures more gracefully, showing a bigger icon of the social media icon rather than a broken image.

#### Changes proposed in this Pull Request:

* Remove the profile picture when it fails to load.
* Show a bigger social icon when the image fails to load.

#### Jetpack product discussion


p1647960248046389-slack-C02JJ910CNL
p1648823351918689-slack-C02JJ910CNL

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure you have one or multiple Publicize connections.
* Go to the "Add New Post" screen and ensure the pictures load correctly.
* If you want to simulate a 404, you can follow these steps in the code:
    * Remove `profilePicture` from the `props`: `const { id, serviceName, label } = props;`
    * Add the following line underneath: `const profilePicture = 'https://jetpack.com/broken404image.jpg'`
* Make sure the image doesn't show, and that you can see a bigger icon instead.

<table>
<tr>
	<td>Before
	<td>After
<tr>
	<td><img width="279" alt="image" src="https://user-images.githubusercontent.com/1713699/173543505-4faa58d2-a683-4642-b973-0f3bb7f50d4a.png">
	<td><img width="280" alt="image" src="https://user-images.githubusercontent.com/1713699/173543524-526aa177-3601-4b7c-9fdb-729544faba21.png">
</table>
